.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
do we still need this file? this was important only when you were writting in CPP right, can it be removed?

Reply:
No, I deleted it